pkg/tfgen/test_data/regress-611-renames.json

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
I guess we will have this file now available in the repo and linked into the provider binary? Just a note that we will need to be mindful to support rolling this out across the providers as part of a rollout for tf bridge. Might involve some ci-mgmt changes etc.